### ISSUE TITLE
Stub out how multi-OS support looks

### ISSUE THREAD

**Issue by kdunn-pivotal** [andreasscherbaum/gpdb-ansible#6]
Again, love what you've done [here](https://github.com/andreasscherbaum/gpdb-ansible/blob/master/roles/gpdb5-dev/tasks/main.yml#L9) - curious to see how the switch/case block fans out when using these detection capabilities.

**Comment by andreasscherbaum** [comment]
This certainly needs to be improved when support for more OS types is added.
Right now I check on the top, so I don't have to add checks down the road in the Playbook.

**Comment by kdunn-pivotal** [comment]
Yep, seems like the right thing to do at this point. Ideally, the playbook would "key" off of the detected OS type. I'm sure there's an example of this somewhere.... Googling...

https://raymii.org/s/tutorials/Ansible_-_Only_if_on_specific_distribution_or_distribution_version.html

**Comment by andreasscherbaum** [comment]
That's what main.yml is doing. One could spread it out even further, and have different files for AWS and GPC and such, but they are all similar enough that it should be enough to just detect the platform it is running on.
